### ISSUE TITLE
PHP - Send tasks for missing alt text to CLI upon attachment upload hooks

### ISSUE THREAD

**Issue by AnthonyLedesma** [the-another/aiccessibility#36]
We should detect cases of fresh uploads and queue those for the AI to generate with CLI utility.
